tests/e2e/provisioning/pkg/client/broker/dto.go

```diff
@@ -0,0 +1,24 @@
+package broker
+
+type ersContext struct {
+	TenantID        string `json:"tenant_id"`
```

Comment:
This filed **should not be used anywhere** because we do not have currently any contract on that filed, so to make it simpler please remove it.

Reply:
removed